test-framework/junit5/src/main/java/io/quarkus/test/junit/IntegrationTestUtil.java

```diff
@@ -296,6 +305,23 @@ public void accept(String s, String s2) {
         return new DefaultDevServicesLaunchResult(propertyMap, networkId, curatedApplication);
     }
 
+    /**
+     * Responsible for activating logging of the tests themselves.
+     * This is a very basic implementation that does not take into account the Quarkus configuration and instead
+     * just uses the same defaults as Quarkus.
+     */
+    static void activateLogging() {
```

Comment:
I think you should be able to use https://github.com/stuartwdouglas/quarkus/blob/a6ad04dadc1a8cb0e64641f0a048e3e6c70b1960/core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java#L62 here. 

It will mostly setup logging as per the config, although if you were doing something odd like pulling log config from a vault it won't work.

Reply:
Good idea! Fixed